pygfx/resources/_texture.py

```diff
@@ -23,7 +23,9 @@ class Texture(Resource):
             (e.g. from ``wgpu.TextureFormat``).
     """
 
-    def __init__(self, data=None, *, dim, size=None, format=None):
+    def __init__(
+        self, data=None, *, dim, size=None, format=None, gamma_correction=0.45
+    ):
```

Comment:
This is a leftover from the previous attemp. Will remove it - this module should remain unchanged.